simtools/ray_tracing.py

```diff
@@ -75,7 +75,7 @@ def __init__(
     ):
         """Initialize RayTracing class."""
         self._logger = logging.getLogger(__name__)
-        self._logger.debug("Initializing RayTracing class")
+        self._logger.info("Initializing RayTracing class")
```

Comment:
I would suggest to keep this on the debug level. We don't need to know each time with initialize RayTracing that this is happening. This is debug information.

Reply:
Okey, set them now back to debug, also the following three you commented on.